src/lib/bootloader/grub2_widgets.rb

```diff
@@ -113,30 +113,46 @@ def store
   end
 
   # Represents decision if smt is enabled
-  class Smt < CWM::CheckBox
+  class CpuMitigations < CWM::ComboBox
     include Grub2Widget
 
     def initialize
       textdomain "bootloader"
     end
 
     def label
-      _("Disable Simultaneous &Multithreading")
+      _("CPU Mitigations")
+    end
+
+    def items
+      [
+        [:nosmt, _("Auto + No SMT")],
+        [:auto, _("Keep SMT")],
+        [:off, _("Off")],
+        [:manual, _("Manually")]
+      ]
     end
 
     def help
+      # TODO: adapt
       _(
-        "<p><b>Disable Simultaneous Multithreading</b><br>\n" \
-          "To disable sharing physical cores by more virtual ones."
+        "<p><b>CPU Speculation</b><br>\n" \
```

Comment:
The title here is "CPU Speculation" but the widget uses the "CPU Mitigations" label. Which one is correct?